tools/webpack/packages.js

```diff
@@ -63,7 +63,7 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 	buildTarget = buildTarget  + '/wp-includes';
 
 	const WORDPRESS_NAMESPACE = '@wordpress/';
-	const BUNDLED_PACKAGES = [ '@wordpress/icons' ];
+	const BUNDLED_PACKAGES = [ '@wordpress/icons', '@wordpress/interface' ];
```

Comment:
@youknowriad, can you confirm this change is expected?

Reply:
oh yes, I'm actually concerned this is done only now :). Did we leak this package to the public API or something?

Reply:
It wasn't listed in `package.json` so probably not, but I can't make it work with the version selected.

Reply:
Indeed, it's surprising, since it probably was replaced with `wp.interface` that doesn't exist ...

Reply:
So it works because `@wordpress/interface` isn't considered an external library in the webpack plugin:

https://github.com/WordPress/gutenberg/blob/8d4eef89b7caaea5db569a2e3c8fd66c7946631f/packages/dependency-extraction-webpack-plugin/lib/util.js#L2